docs/ops/deployment/hadoop.zh.md

```diff
@@ -120,4 +88,44 @@ This way it should work both in local and cluster run where the provided depende
 To run or debug an application in IntelliJ Idea the provided dependencies can be included to the class path
 in the "Run|Edit Configurations" window.
 
+
+2) Putting the required jar files into /lib directory of the Flink distribution
```

Comment:
chinese version wasn't updated

Reply:
Will do that before merging or with the next batch of review comments.